testsupport/wait/host.go

```diff
@@ -1849,6 +1849,19 @@ func UntilSpaceHasTargetClusterRoles(expected []string) SpaceWaitCriterion {
 	}
 }
 
+// UntilSpaceHasTargetClusterRoles returns a `SpaceWaitCriterion` which checks that the given
+// Space has the expected Spec.DisableInheritance value
+func UntilSpaceHasDisableInheritance(expected bool) SpaceWaitCriterion {
+	return SpaceWaitCriterion{
+		Match: func(actual *toolchainv1alpha1.Space) bool {
+			return reflect.DeepEqual(actual.Spec.DisableInheritance, expected)
```

Comment:
as it is a `bool` we may avoid relying on `reflect`

```suggestion
			return actual.Spec.DisableInheritance == expected
```